src/index.ts

```diff
@@ -190,6 +206,12 @@ export default class IncludeFragmentElement extends HTMLElement {
         if (!isWildcard(this.accept) && (!ct || !ct.includes(this.accept ? this.accept : 'text/html'))) {
           throw new Error(`Failed to load resource: expected ${this.accept || 'text/html'} but was ${ct}`)
         }
+        return response
+      })
+      .then(response => {
+        if (cspTrustedTypesCallback) {
+          return cspTrustedTypesCallback(response)
+        }
```

Comment:
This could be `cspTrustedTypesCallback?.(response) ?? response.text()`, but that technically has different behaviour if the callback returns `null` or `undefined` (which would violate its TypeScript signature, but which we can't prevent).